scripts/assign-colors.py

```diff
@@ -1,10 +1,10 @@
+#!/usr/bin/env python3
+"""
+Generate colors.tsv for augur export based on ordering, color schemes, and what exists in the metadata
+"""
 import argparse
 import pandas as pd
 
-# Forced colours MUST NOT appear in the ordering TSV
-forced_colors = {
-}
-
```

Comment:
_non-blocking_

I'd be curious how this was used and whether we'd want to support this as an optional feature.

Reply:
It was always very hacky and used in the early days of ncov to force colours for things like diamond princess (e.g. see https://github.com/nextstrain/ncov/commit/d50745cb5f6dc76bbca098116c5325a31b8732e8). It shouldn't propogate, and hasn't been used in ncov for many years.

Reply:
Ah, thanks for the context!